Harvest.Core/Services/EmailService.cs

```diff
@@ -608,6 +610,24 @@ public async Task<bool> CloseoutConfirmation(Project project)
             }
 
             return true;
+        }
+
+        public async Task<int> ResendCloseoutNotifications()
+        {
+            var count = 0;
+            var projectsAwaitingCloseout = await _dbContext.Projects.Include(a => a.PrincipalInvestigator).Where(a => a.IsActive && a.Status == Project.Statuses.PendingCloseoutApproval).ToArrayAsync();
+            if (projectsAwaitingCloseout.Any())
+            {
+                foreach(var project in projectsAwaitingCloseout)
+                {
+                    if(await CloseoutConfirmation(project, false))

```

Comment:
Don't love error checking by returning true/false but i think the bigger issue is if one send fails it'll fail all of them.  I think a wrap in try/catch within the for loop here would be a good idea

Reply:
@srkirkland  The CloseoutConfirmation() is wrapped in a try catch, but I don't know why all would fail if one fails.

Reply:
ok that'll work